package.json

```diff
@@ -8,7 +8,8 @@
   },
   "scripts": {
     "server": "npx http-server test/",
-    "sass:watch": "npx node-sass test/style.scss test/style.css --watch"
+    "sass:watch": "npx node-sass test/style.scss test/style.css --watch",
+    "sassdoc": "rm -rf dist/ && npx sassdoc src -d dist"
```

Comment:
`rm` is a bash command, so it's not does not work cross-platform - except you use e.g. git bash on windows.

IMO there are a couple of options
1) leave as is, add a notice in the documentation to either use a proper cli or manually delete it
2) use a cross-platform remove tool like [rimraf](https://www.npmjs.com/package/rimraf) or [shelljs](https://github.com/shelljs/shx)
3) do not clean `dist/`. "worst" thing that could happen is leftover assets 

1. People might run into initial troubles
2. Easiest, but added dependency
3. Not clean (haha) but does not break anything

Reply:
Pardon my ignorance towards non-UNIX people.

I think 2. is the most reasonable option.